xbmc/windowing/wayland/XkbcommonKeymap.cpp

```diff
@@ -191,7 +193,7 @@ static const std::map<xkb_keycode_t, XBMCKey> XkbKeycodeXBMCMappings = {
     {XKB_KEY_XF86Phone, XBMCK_PHONE},
     {XKB_KEY_XF86AudioPlay, XBMCK_PLAY},
     {XKB_KEY_XF86AudioRandomPlay, XBMCK_SHUFFLE}
-    // XBMCK_FASTFORWARD clashes with XBMCK_MEDIA_FASTFORWARD
+// XBMCK_FASTFORWARD clashes with XBMCK_MEDIA_FASTFORWARD
```

Comment:
Sorry to nitpick, I saw no other issues in this PR. Can you undo this change, and ignore Jenkins if he wants it back for style?